resources/views/layouts/email.blade.php

```diff
@@ -16,8 +16,8 @@
 
         <div class="footer" style="margin-top: 40px; font-size: 13px; color: #999;">
             <p>Rose Vouchers text</p>
-            <p>For more information please go to <a href="http://www.alexandrarose.org.uk/">www.alexandrarose.org.uk</a> or email us at <a href="mailto:arc@neontribe.co.uk">arc@neontribe.co.uk</a>.</p>
-            <p><a href="http://www.alexandrarose.org.uk/privacy/">Privacy Policy</a></p>
+            <p>For more information please go to <a href="http://www.alexandrarose.org.uk/" role="link">www.alexandrarose.org.uk</a> or email us at <a href="mailto:arc@neontribe.co.uk">arc@neontribe.co.uk</a>.</p>
+            <p><a href="http://www.alexandrarose.org.uk/privacy/" role="link">Privacy Policy</a></p>
```

Comment:
Until a service specific privacy policy is impmeneted, this will do. Card for future update (in the privacy policy card)